fighthealthinsurance/utils.py

```
@@ -399,7 +399,9 @@ async def best_within_timelimit(
         asyncio.create_task(cancel_tasks(list(pending)))
         return best_result_option
     # Ok somehow things are still _bad_
-    raise Exception("No good answers found and we waitied way past the expected time.")
+    raise Exception(
+        "No good answers found and we waitied way past the expected time {timeout}"
+    )
```

Ruff (0.14.8)

Create your own exception

(TRY002)